examples/cpp/gcp_observability/helloworld/README.md

```diff
@@ -0,0 +1,34 @@
+# gRPC C++ GCP Observability Hello World Example
+
+This example consists of a hello world client and a hello world server
+instrumented with GCP Observability for logs, metrics and tracing. Note that
+familiarity with the [basic hello world][] example is assumed.
+
+Please refer to GCP's Microservices Observability user guide for setup
+instructions.
+
+[basic hello world]: https://grpc.io/docs/languages/cpp/quickstart
+
+### Run the example with configuration
+
+To use Observability, you should first setup and configure authorization as
+mentioned in the Microservices Observability user guide.
+
+You need to set the `GRPC_GCP_OBSERVABILITY_CONFIG_FILE` environment variable to
+point to the gRPC GCP Observability configuration file (preferred) or
+alternatively set `GRPC_GCP_OBSERVABILITY_CONFIG` environment variable to gRPC
+GCP Observability configuration value. This is needed by both
+`build/install/example-gcp-observability/bin/gcp-observability-client` and
```

Comment:
This looks like java examples binary PATH, you might need to update this?

Reply:
Yeah, thanks!